src/components/Map/__tests__/__snapshots__/Map.spec.tsx.snap

```diff
@@ -0,0 +1,17 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`<Map/> that renders a Map with markers should render with default props 1`] = `
+<div>
+  <div
+    style="height: 100%; width: 100%;"
+  />
+</div>
+`;
+
+exports[`<Map/> that renders a Map with markers should render with markers 1`] = `
```

Comment:
as discussed elsewhere, there is not much point in these snapshots for this component. I'm ok to leave the first, just the sake of completeness, but this one I'm sure we can remove.

Reply:
:+1: